tools/percolator/msgf2pin.xml

```diff
@@ -1,22 +1,56 @@
-<tool id="msgfplus2pin" name="MS-GF+ output to Pin converter" version="3.0">
+<tool id="msgfplus2pin" name="MS-GF+ output to Pin converter" version="3.0.1">
     <description>to create Percolator input files</description>
     <requirements>
         <requirement type="package" version="3.0">percolator</requirement>
     </requirements>
     <stdio>
         <exit_code range="1:"/>
     </stdio>
-    <command>
-             msgf2pin -o $pinout -e $enzyme
-             #if $matches
-             -m $matches
+    <command><![CDATA[
+             mkdir target && mkdir decoy &&
+             
+             #if $mzids.inputtype == "multi"
+                 #for $key in $mzids.target.keys()
+                     ln -s "$mzids.target[$key]" "target/$key" &&
+                     echo "target/$key" >> targetmeta &&
```

Comment:
Yes and the only reason is that `"` gets evaluated in bash, so it can contains some fancy shell injections ... single quotes do not have this problem.